api/cpp/cmake/SlintConfig.cmake.in

```diff
@@ -13,6 +13,10 @@ endif()
 
 add_library(slint-cpp-shared SHARED IMPORTED)
 set_target_properties(slint-cpp-shared PROPERTIES @SLINT_LIB_PROPERTIES@)
+set_target_properties(slint-cpp-shared PROPERTIES MAP_IMPORTED_CONFIG_RELEASE @CMAKE_BUILD_TYPE@)
+set_target_properties(slint-cpp-shared PROPERTIES MAP_IMPORTED_CONFIG_RELWITHDEBINFO @CMAKE_BUILD_TYPE@)
+set_target_properties(slint-cpp-shared PROPERTIES MAP_IMPORTED_CONFIG_MINSIZEREL @CMAKE_BUILD_TYPE@)
+set_target_properties(slint-cpp-shared PROPERTIES MAP_IMPORTED_CONFIG_DEBUG @CMAKE_BUILD_TYPE@)
```

Comment:
Is it okay if we end up with `MAKE_IMPORTED_CONFIG_RELEASE RELEASE` or do these lines need `if (@CMAKE_BUILD_TYPE@ != $version_in_that_config)` guards?

(maybe this should be written as a `for` loop?)

Reply:
The for loop is a great idea, I'll do that.

I do not think the test for CMAKE_BUILD_TYPE is necessary, but I'll add it anyway as it is not that much code in a for loop:-)